File: Python/Faktöriyel.py
def faktor(n):
    i=1
    sonuc=1
    while i <= n:
        sonuc=sonuc*i
        i=i+1
    print(n,"!=",sonuc)

x = input("Bir Sayi Giriniz: ")
x = int(x)
faktor(x)

File: Pd1 ve Pd2 Örnekler (C)/Faktöriyel.cpp
#include <stdio.h>
#include <conio.h>
int main(void)
{
	int i,sayi,carpim;
	
	carpim=1;
	printf("Bir sayi giriniz : ");
	scanf("%d",&sayi);
	
	for(i=1;i<=sayi;i++){
	
	carpim=carpim*i;
	
	}
	printf("%d! = %d",sayi,carpim);
	getch();
	return 0;
	
}
